vsphere/manifest.json

```diff
@@ -11,7 +11,6 @@
     "mac_os",
     "windows"
   ],
-  "version": "1.0.4",
```

Comment:
`2.0.0` here, right? Or are we removing the check version from the manifest? If we are indeed removing the version from the manifest, we should bump the `manifest_version` field (and document the change somewhere). We do use some of these fields internally (probably not `version` but I'm not sure where we stand).

Reply:
Will bump manifest version, version should be defined only once. Not sure we need to document anything, this field is not used directly anywhere.